internal/commands/push/command_test.go

```diff
@@ -1246,6 +1313,38 @@ func TestPushCommandCreateNewApp(t *testing.T) {
 					},
 					expectedProceed: true,
 				},
+				{
+					description: "should confirm app configuration before creating",
+					procedure: func(c *expect.Console) {
+
+						c.ExpectString("Do you wish to create a new app?")
+						c.SendLine("y")
+
+						c.ExpectString("App Name")
+						c.SendLine("testApp")
+
+						c.ExpectString("App Location")
+						c.Send(string(terminal.KeyArrowDown))
+						c.SendLine("")
+
+						c.ExpectString("App Deployment Model")
+						c.Send(string(terminal.KeyArrowDown))
+						c.SendLine("")
+
+						c.ExpectString("App Environment")
+						c.Send(string(terminal.KeyArrowDown))
+						c.SendLine("")
+
+						c.ExpectString("Are these settings correct?")
+						c.SendLine("")
```

Comment:
remind me, is "blank" here by default saying "No"?  we should update the test case description here to reflect what we're actually doing

Reply:
Yeah, it's a no. Just renamed the test, let me know if the new name looks good.